_posts/2017-12-18-XPDays-Benelux-2017.md

```diff
@@ -0,0 +1,208 @@
+---
+layout: post
+authors: [derya_duru]
+title: 'XP Days Benelux 2017'
+image: /img/xpdays-benelux-2017/XPDays-Benelux-2017.png
+tags: [XP Days Benelux, Agile, Scrum, Scrum Master, Gamification, Integral Theory, Art of Hosting, Facilitation, Coaching]
+category: Conference
+comments: true
+---
+
+> [XP Days Benelux](http://www.xpdaysbenelux.nl){:target="_blank"} is a two day conference on agile software development for and by practitioners.
+In this post, I will go over some talks and sessions I attended and participated in.
```

Comment:
In this post, I will take you along to some talks I attended and some sessions I participated in.